### DISCUSSION TITLE
Please help with the best possible command.

### DISCUSSION THREAD

**Discussion by curiosport** [xinntao/Real-ESRGAN#280] [Q&A]
I have a manga that I want to improve the quality and Real-ESRGAN convinced me because there was a free online service (https://imglarger.com/Anime16K) that claimed to use this API, as it is free it has its limitations of use obviously, so I downloaded the portable version from here to do it on a PC with a GPU Nvidia 2080Ti, I used many commands and I can not imitate the quality of the online service, in the end I'm not sure if that site really uses Real-ESRGAN, maybe I got confused or do not know, since I can not find the link where they claimed that.

Can anyone tell me what is the best possible command to imitate it?

Original, Personal 2080Ti and Server.

**Comment by brunoais** [comment]
Maybe they use Real-ESRGAN but they have different training models
